Assets/MixedRealityToolkit/_Core/Utilities/InputMappingAxisUtility.cs

```diff
@@ -0,0 +1,356 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Utilities
+{
+    /// <summary>
+    /// Utility class for Unity's Input Manager Mappings.
+    /// Input from all types should be defined here for use 
+    /// though out the entire toolkit.
+    /// </summary>
+    public static class InputMappingAxisUtility
```

Comment:
I'm not sure how I feel about using the old input system. I was really hoping to get the input directly from the SDK or driver.

Reply:
Removed the Xbox defaults, as for the InputManager. These work for now until we come up with a better solution later. Start from what works and then iterate.